crates/database/src/jit_orders.rs

```diff
@@ -17,14 +16,23 @@ use {
     },
 };
 
-const JIT_ORDERS_SELECT: &str = r#"
+pub const JIT_ORDERS_SELECT: &str = r#"
```

Comment:
I don't think it's a good idea to leak this query into the general order query. For #2941 you created two functions (one for normal order one for jit orders) and merge the results. This makes sense in my opinion. Why did this approach not work here?

Reply:
For completness adding the same answer as on slack: because of offset, limit and ordering, but mostly offset.
Extracted the user_orders into a separate file